.ci/Jenkinsfile

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env groovy
+@Library('apm@current') _
+
+pipeline {
+  agent any
+  environment {
+    REPO = 'apm-nodejs-http-client'
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    PIPELINE_LOG_LEVEL='INFO'
+    NOTIFY_TO = credentials('notify-to')
+    JOB_GCS_BUCKET = credentials('gcs-bucket')
+    CODECOV_SECRET = "secret/apm-team/ci/${env.REPO}-codecov"
+  }
+  options {
+    timeout(time: 3, unit: 'HOURS')
```

Comment:
I don't think the timeout needs to be anywhere near this high for the http client.

Reply:
10 minutes should be enough. Currently our tests take less than 2 minutes

Reply:
Sure